Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml

```diff
@@ -645,10 +586,10 @@ tasks:
     isautoswitchedtoquietmode: false
   "20":
     id: "20"
-    taskid: 65f0d4d7-46dd-4e48-8f34-864170426cf6
+    taskid: 1edded92-2e34-49d8-802c-c269071f8878
     type: regular
     task:
-      id: 65f0d4d7-46dd-4e48-8f34-864170426cf6
+      id: 1edded92-2e34-49d8-802c-c269071f8878
```

Comment:
Can we look into what we should set for asmcloud.other for Azure a bit, please?  For AWS we use organization and GCP we do folders.  Could add the whole hierarchy info or something else

Reply:
Why would we add that here instead of in `asmassethierarchy`? We could change it up and add subscription name to `other` and tenant name to `organization` from the `AzureAssetHierarchy` output.

Reply:
Yeah at this point we would need to make a change on the `Azure - Enrichmnet` Playbook in the Azure E&R pack because we only get back `azureAssetHierarchy` and is not possible to know what the tenant name is from that (tenant name =/= root management group). Also the `azureAssetHierarchy` key is a path that can be `n` # of management groups.

I'll look into making follow up enrichment tickets to improve Azure Enrichment and make all Query output data from the `Azure - Enrichmnet` Playbook.